manifests/plugin/openscap/params.pp

```diff
@@ -6,4 +6,7 @@
   $listen_on               = 'https'
   $openscap_send_log_file  = '/var/log/foreman-proxy/openscap-send.log'
   $spooldir                = '/var/spool/foreman-proxy/openscap'
+  $contentdir              = '/usr/share/foreman-proxy/openscap/content'
```

Comment:
I'd have thought the full path at all times.  I suggested using the symlink target path though, not the symlink itself?